ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_shared_utils/filtered_tenants.py

```diff
@@ -267,6 +272,10 @@ def filtered_vrfs(self: SharedUtils, tenant: dict) -> list[dict]:
             if vrf["svis"] or vrf["l3_interfaces"] or vrf["loopbacks"] or self.is_forced_vrf(vrf):
                 filtered_vrfs.append(vrf)
 
+            if tenant_evpn_vlan_bundle := get(tenant, "evpn_vlan_bundle"):
```

Comment:
i am not sure that we should do it for SVI too under vrf. 
@xaviramon is this ok if `evpn_vlan_bundle ` set on tenant level than configure VRF's SVI under that bundle.
cc: @ClausHolbechArista

Reply:
It is fine to just have this on the tenant for now. Thanks